lib/util.rb

```diff
@@ -50,8 +50,8 @@ def read
       @hash = JSON.parse(File.read(@path))
       return JSON.parse(File.read(@path))
```

Comment:
Is this a Ruby-ism, to assign the variable to a thing, and then return the thing instead of the variable?

Also, doesn't this actually parse the file *twice*?